cmd/config.go

```diff
@@ -120,17 +116,45 @@ func setNodeCmdRun(cmd *cobra.Command, args []string) error {
 	}
 }
 
-func importCmdRun(cmd *cobra.Command, args []string) error {
-	vms, err := utils.VirtualMachineScaleSetVMsViaKubeconfig()
-	if err != nil {
-		return fmt.Errorf("failed to get VMSS VMs: %w", err)
+func importCmdCommand() *cobra.Command {
+	var subscriptionID string
+	var resourceGroup string
+	var clusterName string
+
+	cmd := &cobra.Command{
+		Use:          "import",
+		Short:        "Import Kubernetes nodes in the configuration",
+		SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			utils.DefaultSpinner.Start()
+			defer utils.DefaultSpinner.Stop()
+			vms, err := utils.VirtualMachineScaleSetVMsViaKubeconfig()
```

Comment:
I was wondering if we should keep this one. I mean, the `VirtualMachineScaleSetVMsViaAzureAPI` works in all the cases, with k8s working or not, so it supersedes `VirtualMachineScaleSetVMsViaKubeconfig`. WDYT?

Reply:
Good Point. Yeah we should do that. But what do you think about `VirtualMachineScaleSetVMsViaKubeconfig` like should it be a fallback mechanism or we should add a flag like `--k8s-api` to allow user to use Kubernetes API if needed? I personally think an extra flag might be a better option?